apps/create-fuels-counter-guide/README.md

```diff
@@ -0,0 +1,23 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-fuels`](https://github.com/fuellabs/fuels-ts).
```

Comment:
Did you mean to reference the repository for `create-fuels`?

Reply:
Same as https://github.com/FuelLabs/fuels-ts/pull/1965#discussion_r1549213016